orchestrate-containers/Ansible Scripts/App_deployment.yml

```diff
@@ -0,0 +1,103 @@
+---
+- hosts: master
+  remote_user: root
+
+  vars_prompt:
+    - name: "MariaPass"
+      prompt: "Please enter the password for MariaDB"
+      private: yes
+      confirm: yes
+
+    - name: "WordPass"
```

Comment:
You mean Wordpress?

Reply:
There should be an `inventory.yml` file. Can you add these variables to the inventory? As an exercise, turn these scripts into Ansible roles.

Reply:
Nope, its a variable name which I use for the Wordpress password.